services/sinner/.goreleaser.yml

```diff
@@ -0,0 +1,93 @@
+project_name: sinner
```

Comment:
This doesn't work https://github.com/synapsecns/sanguine/actions/runs/6641288587/job/18043491419?pr=1380

Make sure we do a [goreleaser] to test these going forward